Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM debian:jessie
+
+# osm2pgsql dependencies
```

Comment:
are all the dependencies listed here osm2pgsql dependencies or are some required for (compiling) osmupdate?